core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystem.java

```diff
@@ -15,19 +15,41 @@
  */
 package com.b2international.snowowl.core.codesystem;
 
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.SortedSet;
+
 import org.hibernate.validator.constraints.NotEmpty;
 
+import com.b2international.commons.http.ExtendedLocale;
+import com.b2international.snowowl.core.branch.Branch;
+import com.b2international.snowowl.core.terminology.TerminologyRegistry;
+import com.b2international.snowowl.core.uri.CodeSystemURI;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.common.collect.ImmutableSortedSet;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
 
 /**
- * Captures metadata about a code system, which holds a set of real-world concepts of medical significance (optionally
- * along with different components forming a description of said concepts) and their corresponding unique code.
+ * Captures metadata about a code system, which holds a set of concepts of
+ * medical significance (optionally with other, supporting components that
+ * together make up the definition of concepts) and their corresponding unique
+ * code.
  */
 @JsonDeserialize(builder=CodeSystem.Builder.class)
 public class CodeSystem {
```

Comment:
Since we are using this now in the Java API, this class should implement `Serializable`.